plugins/scanners/fossid/src/test/kotlin/TestUtils.kt

```diff
@@ -152,7 +152,8 @@ internal fun createConfig(
         urlMappings = null,
         writeToStorage = false,
         logRequests = false,
-        isArchiveMode = isArchiveMode
+        isArchiveMode = isArchiveMode,
+        treatPendingIdentificationsAsError = false
```

Comment:
Consider adding the new option to the [reference.yml](https://github.com/oss-review-toolkit/ort/blob/e4c87afb60ea89e652c9a8ffa1c57896740a6fc9/model/src/main/resources/reference.yml) for documentation purposes.